python/oneseismic/scan/scanners.py

```diff
@@ -206,3 +206,134 @@ def report(self):
             'sample-value-max' : self.maxsample
         }
 
+
+def is_closer(target, point, other):
```

Comment:
Yes, this is only a helper function to that scanner so I can make it a static function. Not sure I agree that more documentation makes sense here since it is only an internal function to be used in the scanner and the intention should be clear. Documenting input requirements would only be needed if someone else was to use it.